k8s/plugin/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "kubectl-plugin"
+description = "Kubectl Plugin"
+version = "1.0.0"
+edition = "2018"
+
+[[bin]]
+name = "kubectl-mayastor"
+path = "src/main.rs"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+openapi = { path = "../../dependencies/control-plane/openapi", default-features = false, features = [ "tower-trace" ] }
+utils = { path = "../../utils/utils-lib" }
+rest-plugin = { path = "../../control-plane/plugin", default-features = false }
```

Comment:
Is this path to the rest-plugin correct? Maybe it should be ../../dependencies/control-plane/plugin